packages/daemons/common.go

```diff
@@ -165,3 +165,14 @@ func getHostPort(h string) string {
 	}
 	return fmt.Sprintf("%s:%d", h, consts.DEFAULT_TCP_PORT)
 }
+
+func getDaemonsToStart() []string {
+	if *conf.IsVDEMasterMode || *conf.IsVDEMode {
```

Comment:
Mabye it is better to do some method in config like IsVDEMode() { return *conf.IsVDEMasterMode || conf.IsVDEMode }